Engines/Wine/QuickScript/QuickScript/script.js

```diff
@@ -87,3 +111,22 @@ QuickScript.prototype.preInstall = function(preInstall) {
     this._preInstall = preInstall;
     return this;
 };
+
+/**
+ * creates shortcut
+ * @param {string} [prefix] prefix name
+ */
+QuickScript.prototype._createShortcut = function(prefix) {
+    var shortcut = new WineShortcut()
+                       .name(this._name)
+                       .type(this._type)
+                       .category(this._category)
+                       .prefix(prefix)
+                       .search(this._executable)
+                       .arguments(this._executableArgs);
+
+    if(this.miniature().isPresent()) {
```

Comment:
Just asking: 
Is it possible to use lambdas in nashorn when calling java functions?
I.e. something like this:
```javascript
this.miniature().ifPresent(miniature => shortcut.miniature(miniature));
```

Reply:
I didn't try but aren't arrow functions ES6? So I'd rather guess it doesn't work.

Reply:
You're right, arrow functions are likely to come with Java 9.
Alternatively we should be able to use `function(miniature) { return shortcut.miniature(miniature); }`.
For more details look here: https://blogs.oracle.com/nashorn/nashorn-and-lambda,-what-the-hey

Reply:
Yes, but honestly I don't think that's better readable than the current solution.